.github/workflows/integration.yml

```diff
@@ -37,6 +34,15 @@ jobs:
           # Instead, leverage `continue-on-error`
           # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error
           #
+          # Failing due to breaking changes in rustfmt 2.0 where empty
+          # match blocks have trailing commas removed
+          # https://github.com/rust-lang/rustfmt/pull/4226
+          - integration: chalk
+            allow-failure: true
+          - integration: crater
+            allow-failure: true
+          - integration: glob
+            allow-failure: true
```

Comment:
These three integration test jobs are failing since rustfmt 2.0 is removing existing trailing commas from empty match blocks after #4226, so adding these to the allowed failure list til after the 2.0 release and the formatting changes are applied upstream:

```bash
 Diff in /tmp/tmp.hWkMNIst66/src/lib.rs:734:
 
                     // Empty match
                     match self.matches_from(follows_separator, file.clone(), i + ti + 1, options) {
-                        SubPatternDoesntMatch => {}, // keep trying
+                        SubPatternDoesntMatch => {} // keep trying
                         m => return m,
                     };
```